samples/excel/01-basics/multiple-property-set.yaml

```diff
@@ -0,0 +1,155 @@
+order: 3
+id: excel-multiple-property-set
+name: Multiple Property Set
+description: Setting multiple properties at once with the rich API object set() method.
+author: OfficeDev
+host: EXCEL
+api_set:
+    ExcelApi: 1.4
+script:
+    content: |
+        $("#setup").click(setup);
+
+		$("#set-multiple-properties-with-object").click(setMultiplePropertiesWithObject);
+		$("#copy-properties-from-range").click(copyPropertiesFromRange);
+
+		async function setMultiplePropertiesWithObject() {
+			try {
+				await Excel.run(async (context) => {
+					const sheet = context.workbook.worksheets.getItem("Sample");
+
+					const range = sheet.getRange("B2:E2");
+					range.set({
+						format: {
+							fill: {
+								color: '#4472C4'
+							},
+							font: {
+								name: 'Verdana',
+								color: 'white'
+							}
+						}
+					})
+					range.format.autofitColumns();
+					await context.sync();
+				});
+			}
+			catch (error) {
+				OfficeHelpers.UI.notify(error);
+				OfficeHelpers.Utilities.log(error);
+			}
+		}
+
+		async function copyPropertiesFromRange() {
+			try {
+				await Excel.run(async (context) => {
+					const sheet = context.workbook.worksheets.getItem("Sample");
+
+					const sourceRange = sheet.getRange("B2:E2");
+					sourceRange.load("format/fill/color, format/font/name, format/font/color");
+					await context.sync();
+
+					const targetRange = sheet.getRange("B7:E7");
+					targetRange.set(sourceRange); 
```

Comment:
It may be worth, via a comment, to highlight how you are now setting object properties based on the loaded-and-synced source range.